utils/datasets.py

```
@@ -0,0 +1,24 @@
+    def __init__(self, name: str, dataframe: pd.DataFrame) -> None:
+        self.name = name
+        self.dataframe = dataframe
+
+
+DATASETS_DIRECTORY = "../datasets/preprocessed"
+
+
+def get_datasets() -> List[Dataset]:
+    datasets = []
+    for filename in filter(
+        lambda f: f.endswith(".parquet"), os.listdir(DATASETS_DIRECTORY)
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)